source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,284 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
+#############################################
+
+Upstream Clear Linux offers a plethora of bundles to choose from.  
+As of version 32570, there are over 1000 bundles and counting.  
+
+To provide you access to even more software outside of the distro, we have 
+enabled two new features to our :command:`mixer` and :command:`swupd` client tools
+starting with release 32570.  
+
+:command:`mixer` can now create 3rd-party bundles out of any content you want 
+without the need to package it into an RPM format first.  
+
+And to consume and manage these bundles, we have added a `3rd-party` 
+subcommand to :command:`swupd`.    
+
+Follow this guide to see how to build an example 3rd-party bundle, set up a web 
+server to host it, install and manage the bundle on a client system. 
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisite
+*************
+
+* Familiarity with :command:`mixer`
+* Familiarity with :command:`swupd` client tool
+
+Create chroot to hold 3rd-party app
+***********************************
+
```

Comment:
Suggest adding a sentence about what a chroot is and how/why swupd uses it.

Reply:
changed all references to "chroot" to "directory".